opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -33,6 +33,9 @@ const rootPrivateQuery = graphql`
         id
         name
       }
+      default_dashboard {
```

Comment:
is that in order to solve the [default dashboard bug](https://github.com/OpenCTI-Platform/opencti/issues/4237)?

Reply:
locally tested 👌

Reply:
Partially yes :)